packages/cli-plugin-scaffold-react-package/template/src/index.tsx

```diff
@@ -3,4 +3,6 @@
  * Reference it from other packages via the following import statement:
  * import { NewComponent } from "PACKAGE_NAME";
  */
-export const NewComponent = function() {};
+export const NewComponent = function() {
+    return Math.random();
```

Comment:
Let's return some kind of DOM here, as the function represents a React component. E.g. 

```js
return <div>This is my new component.</div>;
```